src/core/api_client/tests/factories/mock.factory.ts

```
@@ -0,0 +1,247 @@
+    });
+  }
+
+  static setupTimeoutError(path: string): void {
+    fetchMock.post(AuthFactory.getApiUrl(path), {
+      throws: new ApiClientError('Request Timeout', {
+        kind: 'timeout',
+        temporary: true,
+      }),
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)